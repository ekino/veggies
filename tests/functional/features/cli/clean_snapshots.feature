@cleanSnapshots
Feature: Dummy test for --cleanSnapshots

    Scenario: Should echo salut
        When I run command echo salut
        Then exit code should be 0
        And stdout output should match snapshot
