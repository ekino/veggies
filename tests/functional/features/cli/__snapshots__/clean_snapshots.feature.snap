

exports[`Should echo salut 1.1`] = `"salut
"`;

exports[`Should get deleted 1.1`] = `"salut
"`;
